### DISCUSSION TITLE
Baixando XMLs de NFC-e?

### DISCUSSION THREAD

**Discussion by rodovalho** [TadaSoftware/PyNFe#326] [Q&A] [answered]
Pessoal,

Ao fazer a consulta ao sistema de Distribuição, estou obtendo como resposta apenas as NFs do modelo 55. Porém, gostaria de obter os XMLs de todas, inclusive as NFCe. 

Usando o site da Nota Fiscal Eletrônica (ao invés do WebService) tenho como resposta: "O Modelo informado na chave de acesso é inválido (diferente de 55)."

Sabem dizer como proceder para obter todos os modelos de NFs emitidos para um CNPJ específico?

**Comment by juniortada** [comment, marked as answer]
@rodovalho não existe consulta distribuição para NFC-e, somente NF-e (modelo 55).
